test/integration/gardenlet/shoot/care/care_test.go

```diff
@@ -128,8 +126,6 @@ var _ = Describe("Shoot Care controller tests", func() {
 	})
 
 	JustBeforeEach(func() {
-		DeferCleanup(test.WithFeatureGate(features.DefaultFeatureGate, features.WorkerlessShoots, true))
```

Comment:
Similarly,
https://github.com/gardener/gardener/pull/8417/files#diff-06c3ff8068803257ef890370696c0b75f03870f3cc13036b915e7be6d118f6c2R78
can also be removed

Reply:
And there are more occurences in `pkg/apis/core/validation/shoot_test.go`